vembrane/modules/structured.py

```diff
@@ -170,6 +161,7 @@ def process(
     overwrite_number_format,
     backend,
     variables: Dict[str, Any] | None = None,
+    postprocess: Callable[[Any], Any] | None = None,
```

Comment:
Is the argument of the postprocess function truly of `Any` type?

Reply:
No, should be primitives, dict, list. Fixed in the upcoming commit.